README.md

```diff
@@ -3,62 +3,164 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/18f/cms-hitech-apd/badge.svg?targetFile=web%2Fpackage.json)](https://snyk.io/test/github/18f/cms-hitech-apd?targetFile=web%2Fpackage.json)
 [![Known Vulnerabilities](https://snyk.io/test/github/18f/cms-hitech-apd/badge.svg?targetFile=api%2Fpackage.json)](https://snyk.io/test/github/18f/cms-hitech-apd?targetFile=api%2Fpackage.json)
 
-# CMS HITECH APD app
+# CMS eAPD app
 
 This project aims to create a user-friendly, modern product to allow states to
-submit their Medicaid HITECH APD information to CMS easily and help CMS analysts
-review requests more efficiently.
+submit their Medicaid APD information to CMS easily and help CMS analysts
```

Comment:
This project aims to create a user-friendly, modern product to streamline the creation, submission, review and approval of Medicaid HITECH APDs and their associated contract documents.

What do you think about this?